android/modules/ui/src/java/ti/modules/titanium/ui/TabGroupProxy.java

```diff
@@ -445,14 +445,10 @@ protected void handlePostOpen()
 
 		TabProxy activeTab = handleGetActiveTab();
 		if (activeTab != null) {
-			selectedTab = activeTab;
+			selectedTab = null;
```

Comment:
Are you referring to this `onTabSelected()` method?
https://github.com/appcelerator/titanium_mobile/blob/master/android/modules/ui/src/java/ti/modules/titanium/ui/TabGroupProxy.java#L529
If that is the case, it is intended for the `selectedTab` to be null in order to skip the sending of the `focus/blur` events when a tab is set as an active one before the opening of the TabGroup. But I may have not got what you mean.

Reply:
Oh you're right. Never mind.

Reply:
Note that the unit test around blur events is now failing, and is likely related to the changes in this area...

Reply:
After some digging I found that the "blur" unit test started failing because this PR fixed firing a pair of `focus/blur` events that are not supposed to be fired. That problem was introduced at the same time when the unit test was added (Both by me) and this is the reason it didn't fail every single time in other PRs. With my latest update this should be cleared up.

Reply:
The unit test is passing now.